database_design.txt

```diff
@@ -15,6 +15,7 @@ Fields:
     - first name
     - last name
     - email address
+    - password (FK)
```

Comment:
this should probably be directly a `password hash` field. We don't need to look up the password hash from another table, and salt should not be saved in db since it's supposed to be secret.